### ISSUE TITLE
AttributeError in material.py (Python 3)

### ISSUE THREAD

**Issue by lordi** [pycollada/pycollada#57]
In Python 3, ``itervalues`` does not exist, is it possible to use ``values`` here?

    Traceback (most recent call last):
    (...)
      File "/usr/local/lib/python3.4/dist-packages/collada/material.py", line 374, in load
        for s2d in localscope.itervalues():
    AttributeError: 'dict' object has no attribute 'itervalues'

**Comment by jterrace** [comment]
This was previously fixed:
https://github.com/pycollada/pycollada/commit/ae3f313c65e5ce9bd981adb38d6b07414d41fcae

Are you using a packaged version? I think I haven't released a new version since the fix.

**Comment by lordi** [comment]
Yes, this was the pypi version. Could you release a bug fix version?

**Comment by speleo3** [comment]
this can be closed, release v0.5 fixes it.
